docs/ldml/tr35-keyboards.md

```diff
@@ -280,13 +280,9 @@ This is the top level element. All other elements defined below are under this e
 </keyboard>
 ```
 
-> <small>
->
-> Parents: _none_
-> Children: [version](#Element_version), [~~generation~~](#Element_generation), [info](#Element_info), [names](#Element_names), [settings](#Element_settings), [import](#Element_import), [keyMap](#Element_KeyMap), [displayMap](#Element_DisplayMap), [layer](#Element_layer), [vkeys](#Element_vkeys), [transforms](#Element_transforms), [reorders](#Element_reorder), [backspaces](#Element_backspaces)
-> Occurence: required, single
->
-> </small>
+> <small>Parents: _none_<br/>
+> Children: [version](#Element_version), [~~generation~~](#Element_generation), [info](#Element_info), [names](#Element_names), [settings](#Element_settings), [import](#Element_import), [keyMap](#Element_KeyMap), [displayMap](#Element_DisplayMap), [layer](#Element_layer), [vkeys](#Element_vkeys), [transforms](#Element_transforms), [reorders](#Element_reorder), [backspaces](#Element_backspaces)<br/>
+> Occurence: required, single</small>
```

Comment:
The old markdown generated a single <p> for the block quote, inside a <small> element, but html does not allow <p> inside <small>, only the other way around. Also the old markdown appeared to be intended to generate 3 different lines of block quote for the three different parts (Parents, Children, Occurrence), but that was not happening. So add <br/> to generate the three lines, and move the <small> so it hopefully gets put inside the <p>.

Reply:
this is the way I restructured it in the proposed update (`keyboard-preview` branch) but actually I generate this line automatically now.  I'm going to approve this, but i'd recommend not bothering with the keyboard section because it's just going to create merge conflicts later.

Reply:
This spec is largely rewritten in the branch i mentioned.